### ISSUE TITLE
tMM will not start

### ISSUE THREAD

**Issue by MediaMonger** [tinyMediaManager/tinyMediaManager#130]
__What TMM version are you using?__
Downloaded it today (06JAN17), from the official website, stable release

__release, pre-release, nightly, or directly from GitHub/branch?__
release

__What is the actual behaviour?__
double-click on the tinyMediaManager.exe file and the cursor turns into the "thinking" symbol, lasts about 1-2 seconds then...nothing.
A double-click on the tinyMediaManagerUpd.exe adds some new files to the directory and brings up the launching splash screen where it gets up to 99% then...nothing.

__What is the expected behaviour?__
I expect it to run...start up, show me the list of media files I have, blah, blah, blah...

__Additional__
Have you attached the Logfile from the day it happened? 
there is no log file created...

Your Operating system? (win/mac/linux? version?)
Windows 7, 64bit, running java 8

**Comment by myron0815** [comment]
Just an educated guess:
Do you have an NVidia graphics card with driver 378.49 installed?
If yes, see http://forum.kodi.tv/showthread.php?tid=305019

If no...hmm... hard to say.
Can you upload the launcher.log file somewhere? (or attach it here)

**Comment by MediaMonger** [comment]
Hey myron, 
thanks for the tip...
I did try that NVidia "hotfix" but during the install the program decided I didn't have the hardware that needed to be patched and I do think think it didn't anything...I'll look into it more thoroughly this evening, but...eh?
As for the logs, I'll check again, but I don't think  it even generated any...

**Comment by MediaMonger** [comment]
Hey!
So, you were right on point hoss...after looking into the driver situation a little more in depth, I found that I was in fact running that 378.49 version, however the hotfix link that NVidia put out still didn't seem to apply for my system hardware...anyways, I got tMM to work again by uninstalling the driver. When I rebooted, windows installed the 347.52 version, and now I've got my media manager back so...whatever.
Thanks a lot!
